### PR TITLE
Generate legacy client for ConfigMap

### PR DESCRIPTION
**Pull request by pmorie**
So ready for review

### CODE REVIEW

**Comment by deads2k** [comment]
@pmorie assigned back to you since its not ready.  Reassign when it is.

File: pkg/api/types.go [hunk 1]
```diff
@@ -2152,6 +2152,8 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// +genclient=true
```
Comment (deads2k) [inline pkg/api/types.go]: Is this the only non-generated line?

**Comment by smarterclayton** [comment]
LGTM

File: pkg/api/types.go (hunk 1 above)
Comment (pmorie) [inline pkg/api/types.go]: almost, there's a bug in the client generation that meant I had to write the `ConfigMapExpansion` declaration myself.

Comment (deads2k) [inline pkg/api/types.go]: > almost, there's a bug in the client generation that meant I had to write the ConfigMapExpansion declaration myself.

It looks like that's in the generated code?  What stops it from being stomped by the next regeneration?

**Comment by pmorie** [comment]
Closing, @caesarxuchao already got to it in another PR.

**Pull request closed without merging**